packages/strapi-design-system/src/CarouselInput/CarouselImage.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Box, BoxProps } from '../Box';
+import { Tooltip } from '../Tooltip';
+import { ellipsisStyle } from '../Typography/utils';
+
+export interface CarouselImageProps extends BoxProps<'img'> {
+  alt: string;
+  src: string;
+}
+
+const StyledImage = styled(Box)`
+  ${ellipsisStyle({ ellipsis: true })}
+`;
+
+export const CarouselImage = (props: CarouselImageProps) => {
+  const [isError, setIsError] = useState(false);
+
+  const handleImageError = () => {
+    setIsError(true);
+  };
+
+  if (isError) {
+    return (
+      <Tooltip description={props.alt ?? ''}>
+        <StyledImage as="img" height="100%" {...props} />
+      </Tooltip>
+    );
+  }
```

Comment:
I have no understand why we do this – do you? 😓

Reply:
Nah it's fine, if you don't know it off the top of your head I can try and live with it 🥲